manager/src/main/java/com/redhat/service/bridge/manager/actions/sendtobridge/SendToBridgeActionTransformer.java

```diff
@@ -42,8 +42,7 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
         return transformedAction;
     }
 
-    static String getBridgeWebhookUrl(String bridgeEndpoint) throws MalformedURLException {
-        String fullUrl = String.join("", bridgeEndpoint, bridgeEndpoint.endsWith("/") ? "" : "/", "events");
```

Comment:
Can you please clarify? It doesn't seem to be related with the PR

Reply:
It is not. From https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/502 the `/events` suffix is already present in the endpoint

Reply:
I mean, currently the sendToBridge action is broken on `main` because it sends events to `http://endpoint/events/events`. This is the fix and the tests needed to be changed accordingly

Reply:
Ok thanks for the clarification 👍 

@jiripetrlik @sutaakar 
Do we have a ticket to implement a SendToBridge integration test?

Reply:
@danielezonca we have
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/489
- https://issues.redhat.com/browse/MGDOBR-307